pkg/runtime/k0s/common.go

```diff
@@ -15,16 +15,13 @@
 package k0s
 
 const (
-	RuntimeFlag = "k0s"
-	VersionCmd  = "k0s version"
-
-	DefaultAdminConf = "/var/lib/k0s/pki/admin.conf"
+	DefaultAdminConfPath = "/var/lib/k0s/pki/admin.conf"
 
 	DefaultK0sConfigPath     = "/etc/k0s/k0s.yaml"
 	DefaultK0sWorkerJoin     = "/etc/k0s/worker"
 	DefaultK0sControllerJoin = "/etc/k0s/controller"
 	WorkerRole               = "worker"
 	ControllerRole           = "controller"
 
-	ExternalCRI = "remote:/run/containerd/containerd.sock"
+	ExternalCRIAddress = "remote:/run/containerd/containerd.sock"
```

Comment:
Can this value be modified by configuration, etc.

Reply:
Oh, this would be dynamic config when clusterfile support this or add env spec in clusterfile to support this. Until now, the containerd was installed by sealer default, so this would specify it to this fixed address.